e2e/hooks/hooks.mjs

```diff
@@ -0,0 +1,62 @@
+import { BeforeAll, AfterAll, Before, After, setDefaultTimeout, Status } from '@cucumber/cucumber';
+import playwright from 'playwright';
+import { fixture } from '../fixtures/page.mjs';
+
+let browser;
+let context;
+
+setDefaultTimeout(10000);
+
+BeforeAll(async function () {
+  browser = await playwright.chromium.launch({
+    headless: false,
+    args: ['--disable-web-security', '--allow-running-insecure-content'],
+  });
+});
+
+Before(async function () {
+  context = await browser.newContext({
+    ignoreHTTPSErrors: true,
+    permissions: ['clipboard-read', 'clipboard-write'],
+  });
+  const newPage = await context.newPage();
+  await newPage.goto(`${process.env.PW_BASE_URL}`);
+
+  const networkName = 'devnet';
+  const serviceUrl = 'http://devnet-service.liskdev.net:9901';
```

Comment:
Hmm, somehow i cannot import our global network file to our e2e folder. Tried a few different imports:

import networks, { networkKeys } from 'src/modules/network/configuration/networks';
import networks, { networkKeys } from '../../src/modules/network/configuration/networks';
import networks, { networkKeys } from '@network/configuration/networks';

That way we could use our global network file. I guess another way would be to add it to process.env when we run the project, but from how our current networks is configured it doesn't feel like we want to do that?

Reply:
Ok, strange that you can not import the keys 🤔 but converting them to environment variables can deviate from what we have. 

maybe then for now you can create a global constants that can be imported this e2e which could be reused elsewhere?

Reply:
I think it is because of the .mjs extensions :/. Tried to fix that quick but think it requires more research, dont know if we have time for that now.